services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/diagnosiskeys/DiagnosisKeyBundler.java

```diff
@@ -0,0 +1,150 @@
+package app.coronawarn.server.services.distribution.assembly.diagnosiskeys;
+
+import static app.coronawarn.server.services.distribution.assembly.diagnosiskeys.util.DateTime.ONE_HOUR_INTERVAL_SECONDS;
+import static app.coronawarn.server.services.distribution.assembly.diagnosiskeys.util.DateTime.TEN_MINUTES_INTERVAL_SECONDS;
+import static java.time.ZoneOffset.UTC;
+import static java.util.stream.Collectors.groupingBy;
+
+import app.coronawarn.server.common.persistence.domain.DiagnosisKey;
+import app.coronawarn.server.services.distribution.config.DistributionServiceConfig;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * An instance of this class contains a collection of {@link DiagnosisKey DiagnosisKeys}.
+ */
+public class DiagnosisKeyBundler {
+
+  private final DistributionServiceConfig distributionServiceConfig;
```

Comment:
How about we have fields for the parameters we need and assign them in the constructor, instead of accessing the configuration object at run-time. Might make things a little clearer during debugging and minorly increases readability.

Reply:
and it would get rid of ``Boolean.TRUE``